conductor-config-ci.yaml

```diff
@@ -1,6 +1,21 @@
+#danger_generate_throwaway_device_seed: true
+#dpki:
+#  dna_path: ~
+#  network_seed: wind-tunnel-test
+#  allow_throwaway_random_dpki_agent_key: true
+#  no_dpki: false
```

Comment:
Ideally not removed no. I'd prefer that we can test either way. There's a conversation on #92 about ways to do that.

For now, enabling DPKI would require changing code to pass the config through to common helper functions so they can modify their behaviour around app installs. You can't provide an agent key if you're using DPKI. I'm also not sure that's entirely intentional behaviour around that, but that's another issue.

Basically, more work required yet to support testing with either config.

Reply:
> You can't provide an agent key if you're using DPKI. I'm also not sure that's entirely intentional behaviour around that, but that's another issue.

I agree, that's a confusing behavior that I've also noticed. It will need to be possible for Holo anyway and other people who generate their keys outside of Holochain app installation.